### ISSUE TITLE
Mouse.right_button and Mouse.left_button Discussion

### ISSUE THREAD

**Issue by pathunstrom** [ppb/pursuedpybear#108]
[Suggested in review](https://github.com/ppb/pursuedpybear/pull/107#discussion_r216119047) on #107 perhaps the name of the mouse interface is too wordy?

Discuss potential ways that we can improve these functions.

Suggested alternatives:

* `Mouse.left_button` and `Mouse.right_button`
* `Mouse.left` and `Mouse.right`

Potential considerations:

* One button mice still exist! Should we include error handling for that case?

**Comment by astronouth7303** [comment]
My suggestion would be to use descriptive names, on the basis that left-handed and weird-shaped pointing devices exist. Although the names are not shorter.

* `left` -> `primary`
* `right` -> `secondary`
* (None) -> `tertiary`

As far as the handling of buttons that don't exist: Do we even get information about what mouse buttons are available? I feel like we should not force users to deal with this. (ie, throw an error if they use a basic button that doesn't exist). Maybe we could return `None` instead of `False` if the button is unavailable, and expose the information.

**Comment by jamesBrosnahan** [comment]
Is it a good idea to review the conventions used by a couple GUI libraries and try to conform to their most common naming convention?

**Comment by astronouth7303** [comment]
Closing because #161 happened, which did these things:

* Removed the cached mouse object (`Update.mouse`)
* Changed the button checking syntax to `event.button is ppb.buttons.Primary` or `ppb.buttons.Primary in event.buttons` (depending on the event)
